src/gui/sample.xpm

```diff
@@ -0,0 +1,320 @@
+/* XPM */
```

Comment:
mind explaining what exactly this is?

Reply:
oh yeah ! just started experimenting with this, so XPM is an image format that you can directly include in your C/C++ code ! Si I took an icon file .ico and converted to XPM online, then just did #include "file.xpm" and wxwidget has an API that digest that format. So I use it to show the icon in the system tray. (https://en.wikipedia.org/wiki/X_PixMap I haven't read it yet hehe). I should rename the file name ! so if we have more icon/images in our apps, they all have a meaningful name.